lib/components/FilterPanel/FilterPanelTop.tsx

```diff
@@ -0,0 +1,144 @@
+                  >
+                    {option.label}
+                  </a>
+                </li>
+              ))}
+            </ul>
+          </div>
+        )}
+
+        {/* Right Side Sort option */}
+        {filterOptions && (
+          <div className="ms-auto">
+            <div className="flex flex-wrap items-center gap-3 sm:gap-4">
+              {filterOptions.map((filterOption, index) => (
+                <React.Fragment key={index}>
+                  <div className={DropDownClasses}>
+                    <button
+                      className={DropDownToggleClasses}
+                      type="button"
+                      id={`${filterOption.id}MenuButton`}
+                      data-toggle="dropdown"
+                      aria-haspopup="true"
+                      aria-expanded="false"
+                    >
+                      {filterOption.name}
+                      <DropDownSVG />
+                    </button>
+                    <ul
+                      className="cu-filter__dropdown-menu hidden not-prose list-none absolute z-10 left-0 mt-2 bg-white rounded-md shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none p-3 sm:p-4 w-32 sm:w-40"
+                      aria-labelledby={`${filterOption.id}MenuButton`}
+                    >
+                      {filterOption.options.map((option, optionIndex) => (
+                        <li
+                          className={`flex items-center ${optionIndex !== filterOption.options.length - 1 ? 'mb-4' : ''}`}
+                          key={optionIndex}
+                        >
+                          <input
+                            id={`filter-${option.value}-${index}`}
+                            name={`${filterOption.id}[]`}
+                            type="checkbox"
+                            defaultChecked={option.checked}
+                            className="cu-filter__dropdown-menu-item w-4 h-4 rounded border-cu-black-200 text-cu-red focus:ring-cu-red-100"
+                            value={option.value}
+                            data-label={option.label}
+                          />
+                          <label
+                            htmlFor={`filter-${option.value}-${index}`}
+                            className="pr-6 ml-3 text-sm text-cu-black-600"
+                          >
+                            {option.label}
+                          </label>
+                        </li>
+                      ))}
+                    </ul>
+                  </div>
+                  {index < filterOptions.length - 1 && (
+                    <span className="border-l border-cu-black-100 h-8 w-[1px]"></span>
+                  )}
+                </React.Fragment>
+              ))}
+            </div>
+          </div>
+        )}
+      </div>
+    </div>
+  )
+}
+
+export default Top
```

Comment:
Remove, it's already being exported.

Can be removed, all other components export with the component name higher up.

Also we need to add this for proper naming convention display in storybook

`FilterPanelTop.displayName = 'FilterPanel.Top'`